### PR TITLE
Fix the 1.11 release

### PR DESCRIPTION
**Pull request by vxgmichel**

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -1,12 +1,10 @@
...
 include README.rst
 
-recursive-include tests *
-recursive-exclude * __pycache__
-recursive-exclude * *.py[co]
-
+recursive-include tests *.py *.bat *.sh *.md
 recursive-include docs *.rst conf.py Makefile make.bat
-recursive-include parsec/resources *
+recursive-include parsec/core/gui/forms *.ui
```
Comment (touilleMan) [inline MANIFEST.in]: Nothing about the .sql scripts in `parsec/backend/postgresql` ?

File: setup.cfg [hunk 2]
```diff
@@ -1,4 +1,4 @@
-[wheel]
+[bdist_wheel]
```
Comment (touilleMan) [inline setup.cfg]: What's the difference ?

File: setup.py [hunk 3]
```diff
@@ -366,21 +362,7 @@ def run(self):
...
-                glob.glob("parsec/core/gui/tr/**/*.ts", recursive=True),
-                glob.glob("parsec/core/gui/forms/**/*.ui", recursive=True),
-                glob.glob("parsec/core/gui/rc/**/*.png", recursive=True),
-                glob.glob("parsec/core/gui/rc/**/*.qm", recursive=True),
-                glob.glob("parsec/core/gui/rc/**/*.qrc", recursive=True),
-                glob.glob("parsec/core/gui/rc/**/*.otf", recursive=True),
-            )
-        ),
-    },
+    package_data={"parsec.backend.postgresql": ["init_tables.sql"]},
```
Comment (touilleMan) [inline setup.py]: we may want to add a comment like
```
# Omitting GUI resources given they end up packaged in `parsec/core/gui/_resources_rc.py`
```

File: MANIFEST.in (hunk 1 above)
Comment (vxgmichel) [reply MANIFEST.in]: :+1:

File: setup.cfg (hunk 2 above)
Comment (vxgmichel) [reply setup.cfg]: There was a deprecation warning when running `bdist_wheel`.

File: setup.py (hunk 3 above)
Comment (vxgmichel) [reply setup.py]: :+1:

**Pull request merged**